projects/addon-tablebars/components/table-bars-host/table-bars-host.style.less

```diff
@@ -31,7 +31,7 @@
     position: relative;
     margin: 0 auto;
 
-    &_adaptive {
+    &.t-container_adaptive {
```

Comment:
nit: move it below all media queries and leave as it was.

I am not sure, is it better or not. Just a proposal.

Reply:
I think it's better not to depend on the order. global styles of tui-containers are also done this way